### DISCUSSION TITLE
Local Models

### DISCUSSION THREAD

**Discussion by clickbrain** [bootstrapguru/dexor#7] [General]
This looks really powerful. Thanks for releasing it. Any chance you will make it work with Local Models using an end point with Ollama or other inference servers?

I just won't use centralized AI services. 

Thanks.

**Comment by vijaythecoder** [comment]
Hey, sorry for the late reply. I have added support for Local Models but function calling is a challenge at the moment. I saw Mistral has function calling support, I will add that soon.
